forward/sed.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+from astropy.io import fits
+
+def alphas(z, alpha_0, alpha_1, z1 = 1):
+    """ For a given set of alpha_0, alpha_1 values, return the alpha values for the
+    Dirichlet distribution at the given redshift z, evolving from z=0 to z=z1 (z1 = 1 by default)"""
+
+    return np.power(alpha_0, 1-(z/z1)) * np.power(alpha_1, (z/z1))
```

Comment:
Can you change this to e.g.
`dirichlet_alphas = np.power(alpha_0, 1-(z/z1)) * np.power(alpha_1, (z/z1))`
`return dirichlet_alphas`
as this makes debugging easier.